client/packages/system/src/Item/DetailView/Tabs/ItemVariants/BundledItemVariants.tsx

```diff
@@ -18,6 +12,10 @@ import {
   Link,
   Tooltip,
   Box,
+  MaterialTable,
```

Comment:
I'm noticing that on the Item Variants tab, any changes made by adding/removing/linking variants, etc are not reflected in the UI immediately. Probably needs a query key to be invalidated. 

It's not part of this issue, I see it's already happening on develop, but can you see if it's something quick to fix? If not, can you create an issue for it? Thank